src/rtp/rtcp/twcc.rs

```diff
@@ -1010,7 +1010,7 @@ impl<'a> IntoIterator for &'a TwccSendRegister {
 }
 
 /// Record for a send entry in twcc.
-#[derive(Debug)]
+#[derive(Debug, Copy, Clone)]
```

Comment:
I think this was deliberate. The record is only in one place.

Reply:
This enables an optimisation where we reuse allocations, see `Scratch` `src/packet/bwe/mod.rs`. They are only 64 bytes in size so it didn't feel like a problem to make them copy. 

Maybe `Scratch` is a bit premature though

Reply:
Drop entirely in aa43264